pandasai/helpers/openai_info.py

```diff
@@ -19,6 +19,8 @@
     "gpt-4-turbo-preview-completion": 0.03,
     "gpt-4-0125-preview-completion": 0.03,
     "gpt-4-1106-preview-completion": 0.03,
+    "gpt-4o-completion": 0.015,
+    "gpt-4o-2024-05-13-completion": 0.015,
```

Comment:
This would only account for completion costs, without taking into account prompt's.  Here's a fix:

```suggestion
    "gpt-4o": 0.005,
    "gpt-4o-2024-05-13": 0.005,
    "gpt-4o-completion": 0.015,
    "gpt-4o-2024-05-13-completion": 0.015,
```

Reply:
@nehcneb My bad, I thought this PR was meant to support `gpt-4o` costs "from scratch". All good then :)

Reply:
Thank you for double checking!